Resources/Private/Fusion/Content/Placeholder.fusion

```diff
@@ -1,11 +1,16 @@
-prototype(Networkteam.Neos.Mockup:Placeholder) <  prototype(Neos.Neos:Content) {
-    attributes {
-        class = Neos.Fusion:RawArray {
-            base = 'neos-mockup-placeholder'
-            @process.nodeType >
+prototype(Networkteam.Neos.Mockup:Placeholder) <  prototype(Neos.Neos:ContentComponent) {
+    @context.containerHeight = ${q(node).property('height')}
+
+    renderer = Networkteam.Neos.Mockup:Components.Atoms.Placeholder {
+        attributes = Neos.Fusion:DataStructure {
```

Comment:
a lot of what is done here can also be done in the component itself, especially the style attribute should not be of any concern to the  mapping component